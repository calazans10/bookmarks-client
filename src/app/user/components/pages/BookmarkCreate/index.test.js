import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import renderWithRedux from '../../../../../lib/renderWithRedux';
import { BookmarkCreate } from './index';

describe('BookmarkCreate', () => {
  it('renders without crashing', () => {
    const { container } = renderWithRedux(
      <MemoryRouter>
        <BookmarkCreate onRequestCreateBookmark={jest.fn()} />
      </MemoryRouter>
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
