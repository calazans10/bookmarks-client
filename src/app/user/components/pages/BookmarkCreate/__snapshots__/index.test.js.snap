// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BookmarkCreate renders without crashing 1`] = `
<div
  class="sc-fzXfLS rWSAT"
>
  <div
    class="sc-fzXfLO fuJFRd"
  >
    <p
      class="sc-fzXfLP kvHAaA"
    >
      <span
        class="sc-fzXfLQ ehFqOr"
      />
       
      
    </p>
    <button
      aria-label="Close alert"
      class="sc-fzXfLR iJddGT"
      type="button"
    >
      <span>
        Close
      </span>
    </button>
  </div>
  <header
    class="sc-fzXfLT dkBtqJ"
  >
    <div
      class="sc-fzXfLU biEnOK"
    >
      <div
        class="sc-AykKJ yuoou"
      >
        <a
          class="sc-AykKK BJsjk"
          href="/bookmarks"
        >
          Bookmarks
        </a>
      </div>
      <nav
        class="sc-fzXfLV kVucXT"
      >
        <ul>
          <li
            class="sc-fzXfLW czKXzQ"
          >
            <button
              class="sc-fzXfLX lceogU"
              type="button"
            >
              Sign Out
            </button>
          </li>
        </ul>
      </nav>
    </div>
  </header>
  <main
    class="sc-fzXfMv hgkbuk"
  >
    <form
      autocomplete="off"
      class="sc-AykKC fCHCXg"
      novalidate=""
    >
      <fieldset>
        <legend
          class="sc-AykKD jWsEOj"
        >
          Add bookmark
        </legend>
        <div
          class="sc-AykKE dMuQNO"
        >
          <label
            class="sc-AykKF fboVCh"
            for="title"
          >
            Title
          </label>
          <input
            class="sc-AykKG glNtFx"
            id="title"
            name="title"
            placeholder="Title"
            type="text"
            value=""
          />
        </div>
        <div
          class="sc-AykKE dMuQNO"
        >
          <label
            class="sc-AykKF fboVCh"
            for="url"
          >
            Url
          </label>
          <input
            class="sc-AykKG glNtFx"
            id="url"
            name="url"
            placeholder="Url"
            type="url"
            value=""
          />
        </div>
        <div>
          <button
            class="sc-AykKI bmvfEn"
            type="submit"
          >
            Save
          </button>
          <a
            href="/bookmarks"
          >
            Cancel
          </a>
        </div>
      </fieldset>
    </form>
  </main>
</div>
`;
